### ISSUE TITLE
Message Preview Text needs to be improved

### ISSUE THREAD

**Issue by covrter** [ephemeraHQ/converse-app#1396]

**Comment by covrter** [comment]
let's discuss @alexrisch

**Comment by alexrisch** [comment]
@covrter 
I don't think this is a blocker for our current push, but the text on the conversation list can probably use some love, mostly just feedback for the user:

### Group Update
![Simulator Screenshot - iPhone 15 Pro - 2024-12-19 at 15 40 56](https://github.com/user-attachments/assets/0bf1681f-6b3a-4b2f-bc33-9d215d5a791c)

### Image
![image](https://github.com/user-attachments/assets/42bf7be4-452d-4cd0-b4dc-afdaabfc7f6e)

### Reaction
![image](https://github.com/user-attachments/assets/5e2c16dd-9545-465e-9018-7598be6e2c6b)


### Reply
![image](https://github.com/user-attachments/assets/26f2d818-ff40-4362-8e9b-1feb2851be0a)
![image](https://github.com/user-attachments/assets/755982d0-7391-494b-a9a2-9723927e185a)
